app/javascript/utils/form/final_form/Field.js

```diff
@@ -24,33 +24,70 @@ export const BlockNote = ({ value }) => (
   <span className='checkbox-block_note no-margin padding-left--half'>{value}</span>
 )
 
-export const Input = ({ input, id, meta, type, maxLength, placeholder, ariaLabelledby, fieldName }) => (
+export const Input = ({
+  input,
+  id,
+  meta,
+  type,
+  maxLength,
+  placeholder,
+  ariaLabelledby,
+  fieldName,
+  disabled
+}) => (
   <input {...input}
     id={id || `form-${fieldName}`}
     className={(meta.error && meta.touched && 'error') || ''}
     type={type}
     maxLength={maxLength}
     aria-labelledby={ariaLabelledby}
-    placeholder={placeholder} />
+    placeholder={placeholder}
+    disabled={disabled}
+  />
 )
 
-export const Label = ({ label, fieldName, blockNote, id, labelId, className }) => {
+export const Label = ({ label, fieldName, blockNote, id, labelId }) => {
   return (
     label ? (
-      <label htmlFor={id || `form-${fieldName}`} id={(labelId || `label-${fieldName}`).replace('.', '-')} className={className || 'form-label'}>
+      <label htmlFor={id || `form-${fieldName}`} id={(labelId || `label-${fieldName}`).replace('.', '-')} className='form-label'>
```

Comment:
nit: this is a pretty complicated line, can we move each prop onto its own line?